modules/OCPP/OCPP.cpp

```diff
@@ -652,7 +654,11 @@ void OCPP::ready() {
         const auto upload_logs_response = this->r_system->call_upload_logs(upload_logs_request);
         ocpp::v16::GetLogResponse response;
         if (upload_logs_response.file_name.has_value()) {
-            response.filename.emplace(ocpp::CiString<255>(upload_logs_response.file_name.value()));
+            // we just truncate here since the upload operation could have already been started by the system module and
+            // we cant do much about it, so best we can do is truncate the filename and rather make sure in the system
+            // module that shorter filenames are used
+            response.filename.emplace(
```

Comment:
this could also happen in 2.x, at first glance its conversion function also does not take this into account